crates/lib/Cargo.toml

```
@@ -0,0 +1,75 @@
+[package]
+name = "binstall"
+description = "Library backend for cargo-binstall"
+repository = "https://github.com/cargo-bins/cargo-binstall"
+documentation = "https://docs.rs/binstall"
+version = "0.12.0"
```

Maybe we should set version to `0.1.0` instead?

```suggestion
version = "0.1.0"
```